tests/integration/test_envelope.py

```diff
@@ -145,6 +145,102 @@ def test_strip_measurement_interface(
     assert "measurements" not in event, event
 
 
+def test_ops_breakdowns(mini_sentry, relay_with_processing, transactions_consumer):
+    events_consumer = transactions_consumer()
+
+    relay = relay_with_processing()
+    config = mini_sentry.add_basic_project_config(42)
+    config["config"].setdefault(
+        "breakdowns",
+        {
+            "span_ops": {
+                "type": "span_operations",
+                "matches": ["http", "db", "browser", "resource"],
+            },
+            "span_ops_2": {
+                "type": "span_operations",
+                "matches": ["http", "db", "browser", "resource"],
+            },
+        },
+    )
+
+    transaction_item = generate_transaction_item()
+    transaction_item.update(
+        {
+            "breakdowns": {"span_ops": {"lcp": {"value": 202.1},}},
```

Comment:
Why are you setting this above just to override it here?

Reply:
@k-fish Unsure what you mean. Events might arrive to Relay with breakdown values; this test is to demonstrate that breakdown objects are recursively merged instead of overwritten.

Reply:
Ahhh I see. I didn't know the intent was to test the recursive merge. That makes sense then 👍